README.md

```diff
@@ -1,70 +1,73 @@
-# Getting Started with Create React App
+# Fruitinfo
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Fruitinfo is integrated with Fruityvice API. Fruitinfo is provides data for all kinds of fruit! Users can use Fruitinfo to find out interesting information about fruit and educate yourself.
+## Built With
 
-## Available Scripts
+- React 
+- Redux
+- Sass
+- React-bootstrap 
+- JSX
 
-In the project directory, you can run:
+## Live Demo
+[Live Demo](https://heartfelt-capybara-858cd4.netlify.app/)
```

Comment:
- [ ] Kindly ensure your app is properly deployed. When I visit the webpage, nothing is displayed

  ![jgk](https://user-images.githubusercontent.com/81468675/195873848-818125db-d51c-495e-8e57-08f3b55252f5.jpg)

Reply:
Hello @julie-ify Can I ask you a question on slack?

Reply:
Hi @melashu you can ask the question on the PR

Reply:
I just need advice from you. that why I said WhatsApp or slack